### ISSUE TITLE
Statement not permitted

### ISSUE THREAD

**Issue by garro95** [paperManu/splash#8]
Hi! I'm trying to use splash on Ubuntu 14.04 (Trusty). I installed it but when I try to launch it with "splash --help" it says "Istruzione non consentita (core dump creato)" and doesn't start.
Also when I try to add the blender addon in blender, it give an error: "cannot import name 'ui'".

**Comment by paperManu** [comment]
Hi Garro95,

These are two different issues. Anyway, could you give me your configuration (especially graphic card and video driver). Also, try launching it with GDB and copy the whole stack:

```
gdb --args splash --help
```
Then inside gdb:
```
run
bt full
```

Regarding the Blender plugin, you need the latest Blender version for it to work (2.76). Could you copy the console output so that I can see where is the issue?

**Comment by garro95** [comment]
The output of GDB is pasted here http://paste.ubuntu.com/14241566/

**Comment by paperManu** [comment]
Did you compile Splash yourself, or did you use the package?
If you compiled it, could you rebuild it like this:
```
make clean
CXXFLAGS='-g3 -O0' ./configure
make && sudo make install
```
And then relaunch the whole thing with GDB.

Also please give me some information about your configuration.

**Comment by garro95** [comment]
I got Splash from the repository with apt-get. I'm running Ubuntu 14.04.3 LTS 64bit with the 3.13.0-74-generic version of the Linux kernel.
The hardware is made of an Intel Core 2 Quad Q6600@2.40GHz and a AMG Radeon HD 5770 graphic card with the proprietary drivers.

**Comment by paperManu** [comment]
This looks like the precompiled version uses some newer instructions that your CPU doesn't know about. This is strange though because I don't think I use anything beyond SSE2, but maybe some optimization flags are activating some newer instruction set.

You can always try to compile Splash yourself until I find a fix for this, I keep this issue opened until it is corrected.

**Comment by paperManu** [comment]
This should be corrected in the latest release (0.3.12). Splash was previously built with AVX support by default, although I did not use it directly. I now limit extensions to SSE 4.1, which should make Splash run on 10 years old CPUs.
